lib/PuppeteerSharp/Page.cs

```diff
@@ -456,6 +451,16 @@ private Page(Session client, Target target, FrameTree frameTree, bool ignoreHTTP
             }
         }
 
+        public Task<dynamic> EvaluateAsync(string script, params object[] args)
+        {
+            return _frameManager.MainFrame.EvaluateAsync(script, args);
```

Comment:
convert to an arrow method?

Reply:
If I use an arrow the line gets too long. Instead of breaking into lines I'd rather use braces, what do you think?

Reply:
you could have the arrow on the next line, this way the method is a total of 2 lines instead of 4 lines